### PR TITLE
cutoff prompt when its too long for clip encode

### PR DESCRIPTION
**Pull request by aiXander**
When the input prompt is too long it results in two clip_encoded token vectors of len 77 instead of just one. This causes the `final_conditioning = torch.cat(cond_out, dim=0)` line to crash when other prompts only have one [77,768] token vector.

This pr simply detects when this happens and drops the second encoded part of the input prompt (this effectively ignores the end of the prompt that's too long).

This fixes [this bug](https://github.com/FizzleDorf/ComfyUI_FizzNodes/issues/60)

### CODE REVIEW

**Comment by FizzleDorf** [comment]
after many tests without animatediff, I can assure you this isn't an issue with my nodes but is actually a problem with the AnimateDiff Evolved repo. To change the repo for another's issue isn't the best way forward. This would be better as a bool option for the batch schedulers only. You can test for yourself without animate diff and see that it isn't a problem with this repo.

**Comment by FizzleDorf** [comment]
the actual fix has been added in pr #64 . Thanks for your work, it helped during the size issue errors for a time.
